apps/api/prisma/schema.prisma

```diff
@@ -18,16 +18,78 @@ model Admin {
   updatedAt DateTime @updatedAt
 }
 
+model Interviewer {
+  id                           String                         @id @default(uuid())
+  name                         String
+  disciplines                  Discipline[]
+  interviewSlots               InterviewSlot[]
+  createdAt                    DateTime                       @default(now())
+  updatedAt                    DateTime                       @updatedAt
+  InterviewMemberParticipation InterviewMemberParticipation[]
+}
+
 model Intern {
-  id          String       @id @default(uuid())
-  firstName   String
-  lastName    String
-  email       String       @unique
-  disciplines Discipline[]
-  data        Json
-  interview   Json
-  createdAt   DateTime     @default(now())
-  updatedAt   DateTime     @updatedAt
+  id                String             @id @default(uuid())
+  firstName         String
+  lastName          String
+  email             String             @unique
+  data              Json
+  interviewSlot     InterviewSlot?
+  internDisciplines InternDiscipline[]
+  createdAt         DateTime           @default(now())
+  updatedAt         DateTime           @updatedAt
+}
+
+model InternDiscipline {
+  internId   String
+  intern     Intern           @relation(fields: [internId], references: [id])
+  discipline Discipline
+  status     DisciplineStatus
+  testSlot   TestSlot?        @relation(fields: [testSlotId], references: [id])
+  testSlotId String?
+  testStatus TestStatus?
+  testScore  Int?
+  createdAt  DateTime         @default(now())
+  updatedAt  DateTime         @updatedAt
+
+  @@id([internId, discipline])
+}
+
+model TestSlot {
+  id               String             @id @default(uuid())
+  discipline       Discipline
+  start            DateTime
+  end              DateTime
+  location         String
+  capacity         Int
+  maxPoints        Int
+  InternDiscipline InternDiscipline[]
+  createdAt        DateTime           @default(now())
+  updatedAt        DateTime           @updatedAt
+}
+
+model InterviewSlot {
+  id                           String                         @id @default(uuid())
+  intern                       Intern?                        @relation(fields: [internId], references: [id])
+  internId                     String?                        @unique
+  interviewers                 Interviewer[]
```

Comment:
samo da provjerim, oće ovo interviewers funkcionirat s obzirom na to da je tablica InterviewMemberParticipation  ta koja ih povezuje?

Reply:
mislim da ovo treba maknit i pristupat preko `InterviewMemberParticipation[]`